input/fsh/Clinical datapoints profiles/ListofAliases.fsh

```diff
@@ -0,0 +1,20 @@
+// Profiles HL7 FHIR
+Alias: HL7BodyWeight = http://hl7.org/fhir/StructureDefinition/bodyweight 
+Alias: HL7BodyHeight = http://hl7.org/fhir/StructureDefinition/bodyheight
```

Comment:
HL7 is the organization, so a bit wrong to use the name like this. How about 'Standard' or 'Base'?